python/lsst/pipe/tasks/quickFrameMeasurement.py

```diff
@@ -227,6 +237,42 @@ def _calcMedianXxYy(objData):
         medianYy = np.nanmedian([element['xx'] for element in objData.values()])
         return medianXx, medianYy
 
+    @staticmethod
+    def _getCentreOfMass(exp, nominalCentroid, boxSize):
+        """Get the centre of mass around a point in the image.
+
+        Parameters
+        ----------
+        exp : `lsst.afw.image.Exposure`
+            The exposure in question.
+
```

Comment:
Remove the blank lines between parameters.

Reply:
Ah, didn't realise that was the spec, thanks.